src/client/datascience/commands/activeEditorContext.ts

```diff
@@ -114,9 +115,10 @@ export class ActiveEditorContextService implements IExtensionSingleActivationSer
         if (!this.inNativeNotebookExperiment) {
             return;
         }
-        this.hasNativeNotebookCells
-            .set((this.notebookEditorProvider.activeEditor?.model?.cellCount || 0) > 0)
-            .ignoreErrors();
+
+        // Separate for debugging.
```

Comment:
I'm not really sure what this comment means.

Reply:
Assume that means that rich put this in two steps.
1. Put value into the const
2. Use the const to set the context key
To make it easier to debug by setting a breakpoint on the first of the line. Could be combined into one line a gain. But honestly I find it easier to read as two lines.

Reply:
Oh OK, just wasn't sure if it was meant to be committed this way. But if you find it more readable then this is fine the way it is.